### PR TITLE
Use vendored version of plumbing instead of go get 🦸

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

- Using `go get -d` updates continuously the plumbing dependency, without really updating the vendor folder, which will mean inconsistency.
- Remove unused code that was slowing things down :sweat_smile: 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

### CODE REVIEW

**Comment by chmouel** [comment]
/lgtm
/approve

Thanks 👍🏼🔥

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -20,11 +20,6 @@ set -o pipefail
 
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/library.sh
 
-SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
```
Comment (afrittoli) [inline hack/update-codegen.sh]: I hope this one wasn't slowing us down :P

File: test/dummy_test.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+package test
+
+import (
+	"testing"
+
+	_ "github.com/tektoncd/plumbing/scripts"
+)
+
+func TestDummy(t *testing.T) {
+	t.Log("This is required to make sure we get tektoncd/plumbing in the repository, folder vendor")
```
Comment (afrittoli) [inline test/dummy_test.go]: Nice! :)

File: hack/update-codegen.sh (hunk 1 above)
Comment (vdemeester) [reply hack/update-codegen.sh]: This one wasn't, but ain't used anymore :stuck_out_tongue_closed_eyes:

**Pull request merged**